repos/team.toml

```diff
@@ -0,0 +1,8 @@
+org = "rust-lang"
+name = "team"
+description = "Rust teams structure"
+bots = ["bors", "highfive", "rustbot", "rust-timer"]
```

Comment:
Why are bors and rust-timer in this list? (highfive and rustbot also aren't used, but seem more "OK).

Reply:
This reflects the current state of the things in GitHub. I agree they likely don't need to be there, but perhaps we can make that true by using this mechanism (once the changes to sync-team land).